src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -109,7 +116,22 @@ public override void ConvertLocalPath(IExecutionContext context, string localPat
 
             try
             {
-                ServiceEndpoint vssEndpoint = executionContext.Endpoints.FirstOrDefault(e => string.Equals(e.Name, ServiceEndpoints.SystemVssConnection, StringComparison.OrdinalIgnoreCase));
+                await DownloadArtifacts(executionContext, ReleaseArtifacts, ArtifactsWorkingFolder);
```

Comment:
rename GetArtifactsAsync to DownloadArtifactsAndCommitsAsync?

Reply:
resolved